swo_aws_extension/flows/order.py

```diff
@@ -36,15 +36,16 @@ class InitialAWSContext(BaseContext):
 
     @property
     def mpa_account(self):
-        try:
-            return self.order.get("agreement", {}).get("externalIds", {}).get("vendor", "")
-        except (AttributeError, KeyError, TypeError):
-            return None
+        return self.order.get("agreement", {}).get("externalIds", {}).get("vendor", "")
 
     @property
     def pls_enabled(self):
         return is_partner_led_support_enabled(self.order)
 
+    @property
+    def seller_country(self):
+        return self.order.get("seller", {}).get("address", {}).get("country", "")
```

Comment:
I would be careful with this `get`. Keep in mind that you should distinguish where you may or may not have values. And where values are required. For example, here

seller country here is a required field for extension. When it is not there it is better to raise an exception instead of the pure None. Exception is visible and we can detect that somebody forgot to add select() to the query.

WDYT?